### PR TITLE
add new files

### PR DESCRIPTION
**Pull request by RomanPendyur**

### CODE REVIEW

File: HomeworkProject/index.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/indexStyles.css">
+    <link rel="stylesheet" href="./styles/blogStyles.css">
+    <link rel="stylesheet" href="./styles/contactstyle.css">
+    <title>Document</title>
+</head>
+<body class="body__styles">
```
Comment (roskyba) [inline HomeworkProject/index.html:12]: Rename class

File: HomeworkProject/index.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    <div class="header__styles">
+        <div class="header__logo_style"><p class="header__logo_style-clit">C</p>ahee</div>
+        <div>
+            <ul class="header__nav_style">
+                <li class="header__nav_style-li"><a href="#" class="header__nav_style-a">HOME</a></li>
+                <li class="header__nav_style-li"><a href="#aboutus" class="header__nav_style-a">ABOUT</a></li>
+                <li class="header__nav_style-li"><a href="#" class="header__nav_style-a">SERVICES</a></li>
+                <li class="header__nav_style-li"><a href="#" class="header__nav_style-a">GALLERY</a></li>
```
Comment (roskyba) [inline HomeworkProject/index.html:20]: Maybe it should be `<a href="#gallery">`

File: HomeworkProject/index.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    <title>Document</title>
+</head>
+<body class="body__styles">
+    <div class="header__styles">
+        <div class="header__logo_style"><p class="header__logo_style-clit">C</p>ahee</div>
+        <div>
+            <ul class="header__nav_style">
+                <li class="header__nav_style-li"><a href="#" class="header__nav_style-a">HOME</a></li>
+                <li class="header__nav_style-li"><a href="#aboutus" class="header__nav_style-a">ABOUT</a></li>
+                <li class="header__nav_style-li"><a href="#" class="header__nav_style-a">SERVICES</a></li>
```
Comment (roskyba) [inline HomeworkProject/index.html:19]: Maybe it should be `<a href="#services">`

File: HomeworkProject/index.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+        <div class="fresh_life_right__styles"></div>       
+    </div>
+     <div class="scroll_style">
+       <a href="#top"><img src="./img/scroll.png" alt="scroll"></a>
+    </div>
+    <a name="aboutus"></a><br>
+    <div class="aboutus_panel__style">
+        <div class="aboutus_panel_left__style"></div>
+        <div class="aboutus_panel_right__style">
+            <h2 class="aboutus_panel_right__styles-h2">About Us</h2>
```
Comment (roskyba) [inline HomeworkProject/index.html:41]: Do not use `h2` in class names

File: HomeworkProject/index.html [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+            <p class="aboutus_panel_right__styles-p">
```
Comment (roskyba) [inline HomeworkProject/index.html:42]: Same here. Please, rename and re-check in all your HTML and CSS

**Comment by RomanPendyur** [comment]
Ok, I will try no repeat these errors.

**Pull request closed without merging**